stanford-earth/stanford_research_area
# EARTH-272: Add challenge area checkbox to content type, filmstrip layout to teaser display, and block view.

# READY FOR REVIEW

# Summary
- Adds a "Challenge Area" boolean (checkbox) field to the Research Area content type, sets the "teaser" display to use the filmstrip layout, and creates a view of challenge areas teasers as a block, which can then be added to the home page using the viewfield paragraph type.

# Needed By (Date)
- July 27

# Urgency
- none

# Steps to Test

1. In terminal, cd docroot/modules/stanford/stanford_research_area
2. Checkout this branch, EARTH-272
3. Run drush features-import-all -y
4. Create some research area taxonomy terms and some matching research area pages; include at least a title and a thumbnail image. Check some but not all research area pages as "challenge areas"
5. Add a viewfield paragraph type to a page, and pick the "challenge areas" block to display.
6. Block should display only challenge area teasers (still unformatted).

# Affected versions or modules

# Associated Issues and/or People

# See Also
